rawhttp-core/src/main/java/rawhttp/core/RawHttpOptions.java

```diff
@@ -299,6 +310,24 @@ public Builder allowComments() {
             return this;
         }
 
+        /**
+         * Allows bypassing requirements for CONNECT requests which require host and port to be sent
+         * <p>
+         * RFC-7230 section 5.3.3 defines CONNECT requests must:
+         * <quote>
```

Comment:
Javadoc does not support the full HTML spec.
I think you can do `{@code }` here.

Reply:
Or use `<pre>`, that should work.

Reply:
When you're done, run `gradle javadoc` from the root folder. If that passes, everything should be fine.
Run `gradle check` to run all tests. Also should be fine as I ran locally and it seems fine.